apps/docs/content/guides/platform/supabase-grafana.mdx

```diff
@@ -0,0 +1,246 @@
+---
```

Comment:
Shouldn't this be within the sibling page for metrics?

Reply:
@darora I moved the everything into the metrics page